docs/tools/telemetry/running-telemetry-locally.md

```diff
@@ -29,15 +29,21 @@ Running the elastic search offering locally turns out to be easy.
 
 1. To start collecting information you must run the mymove app locally. To do this navigate to the mymove repo and run the command `make client_run` to start up the local server. You can then navigate through the app to start collecting data.
 
-1. To view this data, on the Kibana page you can click the `Observability` panel and then navigate to the `APM` -> `Services` section on the left hand navigation bar.
+1. To view this data, on the Kibana page you can click the `Observability` panel and then navigate to the `APM` -> `Services` section on the left hand navigation bar. Then click on the `milmove` service.
 
   ![Services Dashboard](/img/telemetry/services-dashboard.png)
 
-  :::info
+## How to find specific transactions
 
-  Services are all named `unknown_service_milmove_gin` and all local transactions besides logging in and logging out are called `server-no-tls`. This is a known [bug](https://dp3.atlassian.net/browse/MB-9926).
+To find specific transactions you can leverage the search bar and utilize some of the metadata added to each transaction.
```

Comment:
This goes back to what is the right verbiage, I think technically we are filtering traces within the server-no-tls transaction(s)?  I don't know if AWS has a first class concept of transactions the way they are in Kibana.  No action needed.

I saw in the otel docs they recommend setting the trace/span name to the http route https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span.  We should probably try conforming to that next.